core/src/main/kotlin/com/alecstrong/sql/psi/core/sql.bnf

```diff
@@ -237,6 +238,9 @@ extension_stmt ::= FAKE_EXTENSION
 multi_column_expr ::= multi_column_expression ( LT | LTE | GT | GTE ) multi_column_expression {
   pin = 2
 }
+multi_column_in_expr ::= multi_column_expression IN [LP expr ( COMMA expr ) * RP | bind_expr] {
```

Comment:
I'm not sure if this should be its own expr or part of `multi_column_expr`

Reply:
If we want to support a single `?`, I prefer its own rule. This makes code gen easier by just looking at the element class. Otherwise you need a special case for `multi_column_expr` in sqldelight.